addons/web/static/src/views/fields/state_selection/state_selection_field.scss

```diff
@@ -0,0 +1,9 @@
+
+.o_form_sheet > .o_field_state_selection {
+    float: right;
```

Comment:
does not seem to work ?
I didn't find any form view where the state selection widget is at the right

Reply:
the commit message says that it was and it is on the left of the sheet :smile: